stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/notifications/internal/service/CallServiceConfig.kt

```diff
@@ -62,6 +62,19 @@ public fun livestreamCallServiceConfig(): CallServiceConfig {
     )
 }
 
+/**
+ * Return a default configuration for the call service configuration for livestream which has no camera
```

Comment:
Wouldn't this type also apply to `audio_rooms`, so the description and name should reflect that?